server/github_pr.go

```diff
@@ -20,13 +21,45 @@ type PullRequestEvent struct {
 	RepositoryURL string              `json:"repository_url"`
 }
 
-type IssueComment struct {
+type EventData struct {
 	Action     string                     `json:"action"`
 	Comment    *github.PullRequestComment `json:"comment"`
 	Issue      *github.Issue              `json:"issue"`
 	Repository *github.Repository         `json:"repository"`
 }
 
+// CheckCLA is true if body contains "/check-cla"
```

Comment:
I would rename these methods to have a "Has" prefix to be more clear. Right now. `e.CheckCLA` indicates this method is checking CLA which is not right.

Something like HasCheckCLA/HasCherryPick/HasAutoAssign/HasUpdateBranch